pkg/util/util.go

```diff
@@ -176,6 +176,21 @@ func FindItem(prefix string, args []string) string {
 	return ""
 }
 
+// ReplaceArgument replace argument value with given value.
+func ReplaceArgument(prefix string, newValue string, args []string, firstOccurrence bool) int {
```

Comment:
I appreciate the idea behind the `firstOccurrence`, but there's no need for it, at least for the moment. If you decide to keep this type of feature, you can follow the idea behind Go's `strings.Replace`, where the last parameter is the max number of times the replace should be applied, with a negative meaning "replace all occurrences".

https://golang.org/src/strings/strings.go?s=23551:23597#L918

Reply:
I'll remove the feature, as you said is not necessary for now.